criu/include/restorer.h

```diff
@@ -162,6 +163,8 @@ struct task_restore_args {
 	siginfo_t			*siginfo;
 	unsigned int			siginfo_n;
 
+	cpu_set_t			*cpualloweds;
```

Comment:
Maybe `allowed_cpus`?

Reply:
> you need to implement a zdtm test.
> Here are our static tests:
> https://github.com/checkpoint-restore/criu/tree/criu-dev/test/zdtm/static
> 
> This is an example of how to run them:
> python test/zdtm.py run -t zdtm/static/env00

OK，I will implement it later, in this week.